include/yaclib/util/detail/atomic_counter.hpp

```diff
@@ -26,6 +26,10 @@ struct AtomicCounter : CounterBase {
     }
   }
 
+  YACLIB_INLINE size_t Get() noexcept {
+    return Get(std::memory_order_relaxed);
+  }
```

Comment:
lets avoid this member. better is explicit memory order
```suggestion
```

Also even if we want to have some default behavior here, it's better to add it for Get(std::memory_order order = std::memory_order_relaxed)